apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/CoreConfiguration.java

```diff
@@ -232,7 +235,37 @@
             WildcardMatcher.valueOf("set-cookie")
         ));
 
-    private final ConfigurationOption<Collection<String>> disabledInstrumentations = ConfigurationOption.stringsOption()
+    private final ConfigurationOption<Collection<String>> enabledInstrumentations = ConfigurationOption.stringsOption()
+        .key("enable_instrumentations")
+        .configurationCategory(CORE_CATEGORY)
+        .description("A list of instrumentations which should be enabled.\n" +
+            "Valid options are ${allInstrumentationGroupNames}.\n" +
+            "An instrumentation is enabled if this configuration is empty or contains the instrumentation and the instrumentation is not disabled.\n" +
+            "\n" +
```

Comment:
```suggestion
        .description("A list of instrumentations which should be selectively enabled.\n" +
            "Valid options are ${allInstrumentationGroupNames}.\n" +
            "When set to non-empty value, only listed instrumentations will be enabled if they are not disabled through <<config-disable-instrumentations>> or <<config-enable-experimental-instrumentations>>.\n" +
            "When not set or empty (default), all instrumentations enabled by default will be enabled unless they are disabled through <<config-disable-instrumentations>> or <<config-enable-experimental-instrumentations>>.\n" +
            "\n" +
```

Try to make it extra-clear that setting this parameter might not be enough, for example with instrumentations that are experimental.
Also, this is not just a simple "toggle instrumentation on" switch at it ONLY enables the listed instrumentations and disables all the others. I wonder if using a different configuration name could help here @felixbarny any idea ?